src/routes/artist.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const {postArtist, getArtists, getArtistById, updateArtistById, patchArtistById, deleteArtistById} = require('../controllers/artist')
+const artistRouter = express.Router()
+
+artistRouter.post('/artists', postArtist)
+
+artistRouter.get('/artists', getArtists)
+
+artistRouter.get('/artists/:id', getArtistById)
+
+artistRouter.put('/artists/:id', updateArtistById)
+
+artistRouter.patch('/artists/:id', patchArtistById)
+
+artistRouter.delete('/artists/:id', deleteArtistById)
```

Comment:
Following on from the comments in app.js the base route of `/artists` should be added there. Here you only need anything that comes after `/artists`
```
artistRouter.post('/', postArtist)

artistRouter.get('/', getArtists)

artistRouter.get('/:id', getArtistById)

artistRouter.put('/:id', updateArtistById)

artistRouter.patch('/:id', patchArtistById)

artistRouter.delete('/:id', deleteArtistById)
```